packages/core/useDraggable/index.ts

```diff
@@ -135,9 +141,16 @@ export function useDraggable(target: MaybeComputedRef<HTMLElement | SVGElement |
   }
 
   if (isClient) {
-    useEventListener(draggingHandle, 'pointerdown', start, true)
-    useEventListener(draggingElement, 'pointermove', move, true)
-    useEventListener(draggingElement, 'pointerup', end, true)
+    let cleanups: Fn[] = []
+    const useCapture = computed(() => resolveUnref(options?.useCapture ?? true))
+    watch(useCapture, (useCapture = true) => {
+      cleanups.forEach(cleanup => cleanup())
+      cleanups = [
+        useEventListener(draggingHandle, 'pointerdown', start, useCapture),
+        useEventListener(draggingElement, 'pointermove', move, useCapture),
+        useEventListener(draggingElement, 'pointerup', end, useCapture),
+      ]
+    }, { immediate: true })
```

Comment:
I think we could make `useEventListener` able to accept ref for the options, so we don't need the workaround here.